src/server/routers/rating.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const {
+    getRatings,
+    postRating
+} = require("../controllers/rating");
+
+const router = express.Router()
+
+router.get("/", getRatings)
+
+router.post("/", postRating)
+
+module.exports = router;
```

Comment:
Other than the URL being visible this looks great!

Reply:
Thanks!